hazelcast/src/main/java/com/hazelcast/client/impl/protocol/codec/builtin/CustomTypeFactory.java

```diff
@@ -166,4 +167,10 @@ public static IndexConfig createIndexConfig(String name, int type, List<String>
 
         return new IndexConfig().setName(name).setType(type0).setAttributes(attributes);
     }
+
+    public static ClientBwListEntryDTO createClientBwListEntry(int type, String value) {
+        // TODO handle null case???
```

Comment:
What error does MC get back currently if this type is missing?

Reply:
I've fixed this TODO. We should get an exception on MC (Java client side) and probably will be `HazelcastException` or one of its childs. I'm going to check the exact behavior when I'll be working on MC PR.